src/sagemaker_containers/beta/framework/__init__.py

```diff
@@ -19,6 +19,7 @@
 from sagemaker_containers import _errors as errors
 from sagemaker_containers import _env as env
 from sagemaker_containers import _functions as functions
+from sagemaker_containers import _logging as logging
 from sagemaker_containers import _mapping as mapping
 from sagemaker_containers import _modules as modules
```

Comment:
change is out of scope for this PR, but its weird that we renamed all these modules _ and then alias them back to the old public name.